src/platforms/windows/windowscaptiveportaldetectionthread.cpp

```diff
@@ -1,131 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "windowscaptiveportaldetectionthread.h"
-#include "leakdetector.h"
-#include "logger.h"
-#include "windowscommons.h"
-
-#include <QScopeGuard>
-
-#include <windows.h>
-
-namespace {
-Logger logger(LOG_WINDOWS, "WindowsCaptivePortalDetectionThread");
-HMODULE s_tunnelLib;
-
-typedef struct {
-  const char* p;
-  size_t n;
-} gostring_t;
-
-typedef int32_t WireguardTestOutsideConnectivity(gostring_t ip, gostring_t host,
-                                                 gostring_t url,
-                                                 gostring_t expectedTestResult);
-
-WireguardTestOutsideConnectivity* s_captivePortalDetection = nullptr;
-}  // namespace
-
-static void tunnelLoggerFunc(int level, const char* msg) {
-  Q_UNUSED(level);
-  logger.log() << "tunnel.dll:" << msg;
-}
-
-static bool loadLibrary() {
-  Q_ASSERT(!s_tunnelLib);
-
-  // This process will be used by the wireguard tunnel the whole time. No need
-  // to call FreeLibrary.
-  s_tunnelLib = LoadLibrary(TEXT("tunnel.dll"));
-  if (!s_tunnelLib) {
-    WindowsCommons::windowsLog("Failed to load tunnel.dll");
-    return false;
-  }
-
-  typedef void (*logFunc)(int level, const char* msg);
-  typedef void WireGuardTunnelLogger(logFunc func);
-
-  WireGuardTunnelLogger* tunnelLogger = (WireGuardTunnelLogger*)GetProcAddress(
-      s_tunnelLib, "WireGuardTunnelLogger");
-  if (!tunnelLogger) {
-    WindowsCommons::windowsLog("Failed to get WireGuardTunnelLogger function");
-    return false;
-  }
-
-  tunnelLogger(tunnelLoggerFunc);
-
-  s_captivePortalDetection = (WireguardTestOutsideConnectivity*)GetProcAddress(
-      s_tunnelLib, "WireguardTestOutsideConnectivity");
```

Comment:
you have to remove https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/windows/tunnel/main.go#L108